tw-tkms-starter/src/main/java/com/transferwise/kafka/tkms/EnvironmentValidator.java

```diff
@@ -1,27 +1,21 @@
 package com.transferwise.kafka.tkms;
 
-import com.transferwise.kafka.tkms.config.TkmsProperties;
-import com.vdurmont.semver4j.Semver;
-import org.apache.commons.lang3.StringUtils;
+import jakarta.validation.ConstraintViolationException;
+import jakarta.validation.Validator;
 import org.springframework.beans.factory.annotation.Autowired;
 
-public class EnvironmentValidator implements IEnvironmentValidator {
+public class EnvironmentValidator extends BaseEnvironmentValidator {
```

Comment:
~~I'm slightly confused with the validation architecture.~~
~~I see we have an interface that is exposing `validate()` method.~~

I got it. It's jakarta / javax validation peculiarities. And one bean is for jakarta, Legacy one for javax. I would probably name them that way, to avoid confusion.

Reply:
I was thinking about that, but then thought that the "Legacy" bean will be deleted this year, hopefully anyway.